todo/serializers.py

```diff
@@ -21,7 +20,9 @@ def validate_tags(self, value):
         return value
 
     def to_representation(self, instance):
-        data = super().to_representation(instance)
+        data = super().to_representation(
+            instance
+        )
         data["tags"] = TagSerializer(instance.tags.all(), many=True).data
```

Comment:
Please use method serializer instead of using to_representation. This is a better way to do things when using DRF